tests/unit/components/input-with-icon/InputWithIcon.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import { screen, render } from '@testing-library/react'
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+import CloudUploadIcon from '@mui/icons-material/CloudUpload'
+
+describe('Input With Icon component', () => {
+  const props = {
+    value: 'TEST',
+    startIcon: <CloudUploadIcon /> // random test icon
```

Comment:
```suggestion
    startIcon: <CloudUploadIcon />
```
What about the comment rule in this project? I think they are forbidden.

Reply:
Yes, I'm sorry
I left a comment to explain the solution. 
and I'm deleting it now.